drivers/ec_host_cmd_periph/Kconfig

```diff
@@ -17,4 +18,21 @@ config EC_HOST_CMD_SIMULATOR
 	help
 	  Enable the EC host command simulator.
 
+config EC_HOST_CMD_PERIPH_ESPI
+	bool "Host commands support using eSPI bus"
+	depends on ESPI_PERIPHERAL_EC_HOST_CMD
+	depends on ESPI_PERIPHERAL_CUSTOM_OPCODE
+	help
+	  Enable support for Embedded Controller host commands using
+	  the eSPI bus
+
+config EC_HOST_CMD_PERIPH_ESPI_SHM_SIZE
+	int "Size of shared memory buffer for eSPI peripheral"
+	default 256
+	depends on EC_HOST_CMD_PERIPH_ESPI
+	help
+	  Size of the shared memory buffer used for receiving and sending
+	  the host commands parapeters. It should have the same size defined
+	  for the EC and AP.
```

Comment:
What about adding an API to the espi driver interface to query the configured size?  `ec_host_cmd_periph_espi.c` doesn't need to allocate any storage based on this size, it just needs to know the size.

Reply:
That sounds better. Maybe instead of adding the function to API, is it possible to add lpc_peripheral_opcode?
There is `ECUSTOM_HOST_CMD_GET_PARAM_MEMORY` so maybe `ECUSTOM_HOST_CMD_GET_PARAM_MEMORY_SIZE`?
It would be a smaller change and will not require to resize the syscalls, adding verify function, etc?